algo/src/main/java/com/hartwig/serve/sources/ckb/ActionableEvidenceFactory.java

```diff
@@ -100,43 +99,52 @@ public Set<ActionableEntry> create(@NotNull CkbEntry entry, @NotNull String sour
                     sourceUrls.add("https://ckbhome.jax.org/profileResponse/advancedEvidenceFind?molecularProfileId=" + entry.profileId());
 
                     Set<String> sourceRelevantTreatmentApproaches = Sets.newHashSet();
+                    Set<String> drugClass = Sets.newHashSet();
```

Comment:
These variable names should be plural, `drugClasses` and `therapies`

Reply:
done